src/SUMMARY.md

```diff
@@ -24,6 +24,7 @@
   - [Compile Time Guarantees](why-rust/compile-time.md)
   - [Runtime Guarantees](why-rust/runtime.md)
   - [Modern Features](why-rust/modern.md)
+  - [An Example in C](why-rust/an-example-in-c.md)
```

Comment:
I would like to start the section with the example. I think it will work very well as a fun and provocative example:

```suggestion
  - [An Example in C](why-rust/an-example-in-c.md)
  - [Compile Time Guarantees](why-rust/compile-time.md)
  - [Runtime Guarantees](why-rust/runtime.md)
  - [Modern Features](why-rust/modern.md)
```

Reply:
> I would like to start the section with the example. I think it will work very well as a fun and provocative example:

As you like it.
Feel free to even rename the section "A Provocative Example in C" or something like that. Honestly, no fitting adjective came to mind when I had to think about the section title.